servers/src/main/java/tachyon/worker/netty/DataServerHandler.java

```diff
@@ -160,6 +167,7 @@ private DataBuffer getDataBuffer(RPCBlockRequest req, BlockHandler handler, long
           return new DataFileChannel((FileChannel) handler.getChannel(), req.getOffset(),
               readLength);
         }
+        // TODO: Implement close
```

Comment:
Does this mean BlockReader does not implement close()? If so, this TODO should probably be in BlockReader, and not here.